drivers/SmartThings/sonos/src/ssdp.lua

```diff
@@ -10,8 +10,8 @@ local SSDP = {}
 local function process_response(val)
   local info = {}
   val = string.gsub(val, "HTTP/1.1 200 OK\r\n", "", 1)
-  for k, v in string.gmatch(val, "([%g]+): ([%g ]*)\r\n") do
-    info[string.lower(k)] = v
+  for k, v in string.gmatch(val, "(.-):(.-)\r\n") do
```

Comment:
This looks like it will now capture the space after the colon in the second group, where previously the leading space would be omitted.

Reply:
Correct, but it's trimmed [here](https://github.com/SmartThingsCommunity/SmartThingsEdgeDrivers/pull/1108/files#diff-20c5b6b201061a964f9e0ffcf0e8bee4b2a5ad1eaa2579e391c566abc87b7672R14)

Reply:
It took me a second to get a pattern that worked well with the oddly formed `EXT:\r\n` header, but I've updated the code here to exclude leading whitespace without using a hard-coded single space.